components/GameCard.js

```diff
@@ -21,7 +21,7 @@ const StyledRankTitle = styled.p`
   color: #ff8200;
 `;
 const StyledCard = styled.div`
-  width: 80%;
+  max-width: 80%;
```

Comment:
ah please put the max-width to the "parent" element of this card, most likely the ul element. one example would be: 
```
  margin: 0 auto;
  padding: 0 2rem;
  max-width: 600px;

```

Reply:
and remove the width on this element.

Reply:
const StyledUList = styled.ul`
  max-width: 80%;
  list-style: none;
  display: flex;
  flex-direction: column;
  gap: 20px;
  padding: 0;
  margin: auto;
`;